src/core/InstancedInterleavedBuffer.js

```diff
@@ -1,40 +1,38 @@
 import { InterleavedBuffer } from './InterleavedBuffer.js';
 
-function InstancedInterleavedBuffer( array, stride, meshPerAttribute ) {
+class InstancedInterleavedBuffer extends InterleavedBuffer {
 
-	InterleavedBuffer.call( this, array, stride );
+	constructor( array, stride, meshPerAttribute ) {
 
-	this.meshPerAttribute = meshPerAttribute || 1;
+		super( array, stride );
 
-}
-
-InstancedInterleavedBuffer.prototype = Object.assign( Object.create( InterleavedBuffer.prototype ), {
+		Object.defineProperty( this, 'isInstancedInterleavedBuffer', { value: true } );
 
-	constructor: InstancedInterleavedBuffer,
+		this.meshPerAttribute = meshPerAttribute || 1;
```

Comment:
`|| 1` can be turned into a default parameter.

Reply:
> `|| 1` can be turned into a default parameter. Same for `InstancedBufferAttribute`.

Thanks you, I had change it.